namespace-overview/design.md

```diff
@@ -1,21 +1,18 @@
 # Namespace Overview
 
 ## Summary
-The namespace overview would give a high-level summary of the health of each namespace (or composite) as well as some other basic information.
+The namespace overview would give a high-level summary of the health of the services in each namespace.
 
-## Card View
 ![card view of namespaces](img/namespace-cards.png)
-- Namespaces could be shown with their name and a rollup of their overall health as well as any of several different metrics
 
-## List View
-![list view of namespaces](img/namespace-list.png)
+### Namespace Cards
+- **Title:** Displays the name of the namespace. May wrap to a second line if necessary.
+- **Service Count:** The total number of services in the namespace.
+- **Status Counts:** Shows the number of services with each status. Icons should only be shown for a given status if there are services in the namespace that have that status.
 
-## Alternative Namespace Representations
-![alternative representations of namespaces](img/namespace-alternates.png)
-- Stats we could include
-  - Total number of services
-  - Component pieces of namespace health
-  - Summary of service status
-  - Traffic information
-  - Stats about services with ingresses
-  
+
+### Card interactions
+- Clicking on a card links to a view of the services list filtered by the relevant namespace
```

Comment:
@cshinn this is a bit confusing to me.  The view is called namespaces, the name on the card is namespace, so I think clicking on the card would imply that the user would navigate to the namespace.

Could we instead provide a link on the #Services and allow that to provide the navigation?

Reply:
Is there a plan to include namespaces as objects in Kiali? AFAIK right now they're really only represented as attributes on services

Reply:
@cshinn no, namespaces are not "direct" objects in Kiali. Those are k8s/openshift objects that are used for filtering in Kiali. I don't expect this to change.

Reply:
My point is just that if the view is called namespaces, and we are navigating to services, it should be done so by clicking the "services" link, rather than the card.